index.js

```diff
@@ -18,4 +20,339 @@ const openMenu = document.querySelector('.button');
+    div.innerHTML = `<h2 class="popupname">${arr[4].name.innerHTML}</h2><ul class="popuplist">${arr[4].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[4].image}<div class = "popbox2"><p class = "popuppara">${arr[4].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click5() {
+  if (button5.id == 'probutton5') {
+    div.innerHTML = `<h2 class="popupname">${arr[5].name.innerHTML}</h2><ul class="popuplist">${arr[5].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[5].image}<div class = "popbox2"><p class = "popuppara">${arr[5].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click6() {
+  if (button6.id == 'probutton6') {
+    div.innerHTML = `<h2 class="popupname">${arr[6].name.innerHTML}</h2><ul class="popuplist">${arr[6].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[6].image}<div class = "popbox2"><p class = "popuppara">${arr[6].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+
+button0.addEventListener('click', click0);
+button1.addEventListener('click', click1);
+button2.addEventListener('click', click2);
+button3.addEventListener('click', click3);
+button4.addEventListener('click', click4);
+button5.addEventListener('click', click5);
+button6.addEventListener('click', click6);
+
+function tog() {
+  const togbutton = document.querySelector('#project-button');
+  const showpop = document.querySelector('#content');
+  if (togbutton.style.display !== 'none') {
+    showpop.style.display = 'block';
+    togbutton.style.display = 'none';
+    const list = document.querySelectorAll('.tags');
+    for (let i = 0; i < list.length; i++) {
+      list[i].classList.remove('tags');
+      list[i].classList.add('poptag');
+    }
+  } else if (togbutton.style.display === 'none') {
+    showpop.style.display = 'none';
+    togbutton.style.display = 'block';
+    const list1 = document.querySelectorAll('.poptag');
+    for (let i = 0; i < list1.length; i++) {
+      list1[i].classList.add('tags');
+      list1[i].classList.remove('poptag');
+    }
+  }
+}
+
+const showpopup = document.querySelector('#project-button');
+showpopup.addEventListener('click', tog);
+const showpopup1 = document.querySelector('#probutton');
+showpopup1.addEventListener('click', tog);
+const showpopup2 = document.querySelector('#probutton2');
+showpopup2.addEventListener('click', tog);
+const showpopup3 = document.querySelector('#probutton3');
+showpopup3.addEventListener('click', tog);
+const showpopup4 = document.querySelector('#probutton4');
+showpopup4.addEventListener('click', tog);
+const showpopup5 = document.querySelector('#probutton5');
+showpopup5.addEventListener('click', tog);
+const showpopup6 = document.querySelector('#probutton6');
+showpopup6.addEventListener('click', tog);
```

Comment:
- **[OPTIONAL]** Kindly refactor your code to avoid repetitive code, remember DRY (don't repeat yourself) for the better practices of JavaScript. 👍🏽